src/terra-dev-site/guide/Component Guides/RecreatingItemCollection.guide.mdx

```diff
@@ -0,0 +1,137 @@
+import ResponsiveListandTableExample from '../../example/ResponsiveListAndTableExample?dev-site-example';
+
+# Building an Item Collection
```

Comment:
Rename this file to be the same as the title here. `BuildingAnItemCollection.guide.mdx`

Reply:
Renamed https://github.com/cerner/terra-ui/pull/310/commits/353727590506d9c58f0f44b2cbfd34a9accec9b7